### PR TITLE
Improve summarization of breakpoint capture

### PR DESCRIPTION
**Pull request by matthewloring**

### CODE REVIEW

**Comment by matthewloring** [comment]
@ofrobots PTAL.

File: lib/state.js [hunk 1]
```diff
@@ -437,9 +454,8 @@ StateResolver.prototype.resolveMirrorFast_ = function(mirror) {
 
 StateResolver.prototype.getMirrorProperties_ = function(mirror) {
   var numProperties = this.config_.capture.maxProperties;
-  var namedProperties = mirror.properties(1, numProperties);
-  var indexedProperties = mirror.properties(2, numProperties);
-  return namedProperties.concat(indexedProperties);
+  var properties = mirror.properties();
```
Comment (ofrobots) [inline lib/state.js]: Does this work with all node versions? I am curious why the original code is the way it is.

**Comment by ofrobots** [comment]
LGTM if the .properties() call isn't the cause for the segfault on node 0.12.

File: lib/state.js (hunk 1 above)
Comment (matthewloring) [inline lib/state.js]: This does work as it is only executed on newer versions of node already. The culprit was the call to  `propertyNames` above. Fixed.

**Comment by matthewloring** [comment]
@ofrobots Ready for another look.

**Comment by ofrobots** [comment]
LGTM once the CI is green.

**Pull request merged**